tests/console/yast2_lan_hostname.pm

```diff
@@ -25,9 +25,9 @@ sub hostname_via_dhcp {
     $cmd{hostname_dns_tab} = 'alt-s';
     $cmd{home}             = 'home';
     $cmd{spc}              = 'spc';
-
-    y2logsstep::yast2_console_exec(yast2_module => 'lan');
-    accept_warning_network_manager_default;
+    y2_module_consoletest::yast2_console_exec(yast2_module => 'lan');
+    type_string "Y2DEBUG=1 ZYPP_MEDIA_CURL_DEBUG=1 yast2 \n";
```

Comment:
Correct me if I'm wrong but it seems like the line
```type_string "Y2DEBUG=1 ZYPP_MEDIA_CURL_DEBUG=1 yast2 \n";```

came to your branch after you made a merge. I guess we do not need it here to avoid conflicts while merging.

Reply:
sure